generator/camera_info_collector.cpp

```diff
@@ -317,6 +317,24 @@ void CamerasInfoCollector::Camera::Serialize(FileWriter & writer,
   routing::SerializeSpeedCamera(writer, m_data, prevFeatureId);
 }
 
+bool IsCamerasInfoProhibited(std::string const & mwmName)
+{
+  std::array<std::string, 4> kCountryBlockList = {
```

Comment:
std::array<std::string, 4> constexpr kCountryBlockList

Reply:
you can't apply constexpr to std::string